docs/FAQ.md

```diff
@@ -183,6 +183,28 @@ if novel, ok := resp.Favorite.(*GetBooksFavoriteNovel); ok {
 
 The interface-type's GoDoc will include a list of its implementations, for your convenience.
 
+If you only want to request shared fields of the interface (i.e. no fragments), this may seem like a lot of ceremony.  If you prefer, you can instead add `# @genqlient(struct: true)` to the field, and genqlient will just generate a struct, like it does for GraphQL object types.  For example, given:
+
+```graphql
+query GetBooks {
+  # @genqlient(struct: true)
+  favorite {
+    title
+  }
+}
+```
+
+genqlient will generate just:
+
+```go
+type GetBooksFavoriteBook struct {
+  Title string
+}
+```
+
+Keep in mind that if you later want to add fragments to your selection, the types will change entirely.
```

Comment:
Hmm, if you say `struct: true` and specify fragments, how does it work?  Wouldn't it give an error?

Reply:
Yeah, I just mean it'll error so you'll have to change it.  Will reword.